aws/resource_aws_codepipeline_test.go

```diff
@@ -4,20 +4,20 @@ import (
 	"fmt"
 	"os"
 	"regexp"
+	"strings"
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/codepipeline"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/flatmap"
```

Comment:
We should really get away from using `flatmap`. It was _almost_ completely gone a few months ago, but we wound up with this hard copy in the repository as a compromise to remove Terraform core as a dependency. 😄 See also: https://github.com/terraform-providers/terraform-provider-aws/issues/8902

Instead I would recommend inspecting the `codepipeline.PipelineDeclaration` type directly (if you want to keep this level of testing).